test/internet/test-http-dns-fail.js

```diff
@@ -4,7 +4,6 @@
  * should trigger the error event after each attempt.
  */
 
-var common = require('../common');
```

Comment:
@rvagg I see you added that in https://github.com/nodejs/node/commit/05daf5f7b77e01abf83f25bce859aaa1b1313567, but that PR-URL is 404. Do you think it's possible to remove it so tests that don't utilize `common` wont't have to require it?